### PR TITLE
new package: openvpn 2.4.8

### PR DESCRIPTION
**Pull request by a-schaefers**
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/openvpn/build [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+autoreconf -i
```
Comment (dylanaraps) [inline community/openvpn/build]: Is there no way to avoid `autoreconf`? ie a release tarball of some kind?

Comment (a-schaefers) [reply community/openvpn/build]: When make doesn't work without ./configure, and when there is only a configure.ac file I have to run autoreconf to create configure so we can make the magic happen. If there is a better way let me know??

Comment (konimex) [reply community/openvpn/build]: Generally you'll want to find a better source than directly fetching it from GitHub tags (but still from its official site). Try to check the official OpenVPN site, or find the source from Void's `template` for this package.

File: community/openvpn/sources [hunk 2]
```diff
@@ -0,0 +1 @@
+https://github.com/OpenVPN/openvpn/archive/v2.4.8.tar.gz
```
Comment (dylanaraps) [inline community/openvpn/sources]: Try this link: `https://swupdate.openvpn.org/community/releases/openvpn-2.4.8.tar.xz`

This should remove the `autoreconf` requirement.

Comment (a-schaefers) [reply community/openvpn/sources]: cool!

File: community/openvpn/build (hunk 1 above)
Comment (a-schaefers) [reply community/openvpn/build]: thanks

File: community/openvpn/build [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    --prefix=/usr \
+    --sbindir=/usr/bin \
+    --enable-iproute2 \
+    --enable-pkcs11 \
+    --enable-plugins \
+    --enable-password-save \
+    --enable-x509-alt-username \
+    --disable-plugin-auth-pam
+
+make DESTDIR="$1" install
```
Comment (dylanaraps) [inline community/openvpn/build]: Missing standalone `make`.

**Comment by a-schaefers** [comment]
fixed

**Pull request merged**